<launch>
<param name="n_robot" value="3" />

    <node name="waypoint_assignment" pkg="my_package" type="Waypoint_assignment.py" ns="robot_1">
        <param name="x" value="1" />
    </node>
</launch>